### ISSUE TITLE
Crash on no network connection error

### ISSUE THREAD

**Issue by NHLGames** [NHLGames/NHLGames#139]
The app crashes when a no network connection error occurs (maybe all errors)

In the WriteLine function in the ConsoleRedirectStreamWriter class:

```
System.InvalidOperationException:
Invoke or BeginInvoke cannot be called on a control until the window handle has been created.
```

**Comment by jwallet** [comment]
i had a merged conflict with master, looks like when building the app your VS overwrote some lines in app.config

**Comment by jwallet** [comment]
did you only built the app and launched it, or try different error scenarios ?

**Comment by NHLGames** [comment]
This seems to have fixed the crash.  Another very minor bug:  The version # does not display if you have no connection:

![image](https://user-images.githubusercontent.com/17696817/34440679-c371c154-ec84-11e7-8657-adbd5ac5c9c4.png)

**Comment by jwallet** [comment]
just to be sure, are you trying to simulate a scenario where someone is trying to launch the app without internet connection or you have this issue and your network is connected

**Comment by NHLGames** [comment]
I get this when I run the app on my work network, the proxy is blocking the requests haha.  Usually only HTTP over a web browser is allowed.  The version issue isn't a big deal, just a cosmetic thing really.  I'm assuming the function isn't running until a connection is found to load the games

**Comment by jwallet** [comment]
pretty sure i can display the version at this point.

**Comment by NHLGames** [comment]
Fixed with #141.
